distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector.enso

```diff
@@ -824,8 +825,8 @@ type Vector a
            collection) generates a random set of indices and then selects these
            indices, but we can sample the vector directly.
         Index_Sub_Range.Sample count seed ->
-            Random.set_seed seed
-            Random.item self count
+            rng = Random.new_generator seed
+            rng.item self count
```

Comment:
Shouldn't this be `items` now? I assume some test shall be failing, if it's not it means we don't have enough tests 😅 

```suggestion
            rng.items self count
```

Reply:
Done.